CameraXBasic/app/build.gradle

```diff
@@ -47,17 +47,6 @@ android {
        }
     }
 
-    // Set the source of tests to same for both Unit and Instrumented tests
-    sourceSets {
-        String sharedTestDir = 'src/test/java'
-        test {
-            java.srcDir sharedTestDir
-        }
-        androidTest {
-            java.srcDir sharedTestDir
-        }
-    }
-
```

Comment:
Why is this being deleted? We don't have many tests yet, but the idea is that all tests we write should be compatible with both Robolectric and real devices